jobs/v4/src/main/java/com/example/jobs/JobSearchCreateJob.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.jobs;
+
+// [START job_search_create_job_beta]
+
+import com.google.cloud.talent.v4beta1.CreateJobRequest;
+import com.google.cloud.talent.v4beta1.Job;
+import com.google.cloud.talent.v4beta1.JobServiceClient;
+import com.google.cloud.talent.v4beta1.TenantName;
+import com.google.cloud.talent.v4beta1.TenantOrProjectName;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+public class JobSearchCreateJob {
+
+  public static void createJob() throws IOException {
+    // TODO(developer): Replace these variables before running the sample.
+    String projectId = "your-project-id";
+    String tenantId = "your-tenant-id";
+    String companyId = "your-company-id";
+    String requisitionId = "your-unique-req-id";
+    String title = "your-job-title";
+    String description = "your-job-description";
+    String jobApplicationUrl = "your-job-url";
+    String addressOne = "your-job-address-1";
+    String addressTwo = "your-job-address-2";
+    String languageCode = "your-lang-code";
```

Comment:
Can we inline any of these?

Reply:
What do you mean by that? remove some optional params from method?

Reply:
Like is it really necessary in the sample for these to be passed in?

Couldn't you just put them below: 
```
Job job =
          Job.newBuilder()
              .setCompany(companyId)
              .setRequisitionId(requisitionId)
              .setTitle("job-title")
              .setDescription("description")
              .setApplicationInfo(applicationInfo)
              .addAddresses("your-address")
              .setLanguageCode("en-US")
              .build();
```

Reply:
ahh, okay got it, will make these changes